.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Figured specifying these would be a good move to ensure folks using these starters on going to have a good time.